dencam/buttons.py

```diff
@@ -30,13 +30,13 @@ class ButtonHandler(Thread):
 
     """
 
-    def __init__(self, recorder, state, State_List, stop_flag):
+    def __init__(self, recorder, state, state_list, stop_flag):
         super().__init__()
 
         self.recorder = recorder
         self.state = state
         self.stop_flag = stop_flag
-        self.State_List = State_List
+        self.STATE_LIST = state_list
```

Comment:
the button attribute self.STATE_LIST should be all lowercase so as not to confuse with global variable STATE_LIST